README.md

```diff
@@ -290,6 +290,33 @@ pelias compose up
 pelias test run
 ```
 
+## Optionally cleanup temporary files
+
+Once the build is complete, you can cleanup temporary files that are no longer useful. The numbers in this snippet below are rough estimates for a full planet build.
+
+```
+# These folders can be entirely deleted after the import into elastic search
+rm -rf /data/openaddresses #(~43GB)
+rm -rf /data/tiger #(~13GB)
+rm -rf /data/openstreetmap #(~46GB)
+rm -rf /data/polylines #(~2.7GB)
+
+# Within the content of the "interpolation" folder (~176GB) we must
+# preserve "street.db" (~7GB) and "address.db" (~25GB), the rest can be deleted
+cd /data/interpolation
+rm -rf -- !("street.db"|"address.db")
+
+# Within the content of the "placeholder" folder (~1.4GB), we must
+# preserve the "store.sqlite3" (~0.9GB) file, the rest can be deleted
+cd /data/placeholder
+rm -rf -- !("store.sqlite3")
+
+# Within the content of the "whosonfirst" folder (~30GB), we must
+# preserve the "data" and "meta" folders (~29.5GB)
+cd /data/whosonfirst
+rm -rf -- !("data"|"meta")
```

Comment:
this might also have a SQLite file, I would say nothing in `/data/whosonfirst` can be safely deleted. The downloader removes all temporary files and archive files used during the download process, so only the useful data remains.

Reply:
In my case, the "whosonfirst" folder only has 3 items: "data", "meta" and a file named "whosonfirst-data-postalcode-gb-latest.csv" which is about 582MB.
Do you know if that file used at runtime?
Or should I just remove the "whosonfirst" folder deletion just to be safe?

Reply:
Hmm interesting. That file must have come from an interrupted download. We might be able to prevent that from occurring, but if not I would say that anticipating files like that being present is not worth it, and we should still remove the advice to delete anything in `/data/whosonfirst`

Reply:
OK I see.
I understand it's probably a bad practice to recommend deleting stuff from a folder with only useful resources, so I edited my PR to remove that.

Reply:
Looks good now. Thanks :)